monitoring/local_monitoring/local_monitoring_demo.sh

```diff
@@ -0,0 +1,289 @@
+}
+
+#
+#
+#
+function kruize_local_demo_setup() {
+	# Start all the installs
+	start_time=$(get_date)
+	echo
+	echo "#######################################"
+	echo "#       Kruize Local Demo Setup       #"
+	echo "#######################################"
+	echo
+
+	if [ ${kruize_restart} -eq 0 ]; then
+		clone_repos autotune
+		clone_repos benchmarks
+		if [ ${CLUSTER_TYPE} == "minikube" ]; then
+			minikube >/dev/null
+			check_err "ERROR: minikube not installed"
+			minikube_start
+			prometheus_install autotune
+		fi
+		benchmarks_install
+	fi
+	kruize_local_patch
+	kruize_install
+	echo
+
+	kubectl_cmd="kubectl -n default"
+	TECHEMPOWER_PORT=$(${kubectl_cmd} get svc tfb-qrh-service --no-headers -o=custom-columns=PORT:.spec.ports[*].nodePort)
+	TECHEMPOWER_IP=$(${kubectl_cmd} get pods -l=app=tfb-qrh-deployment -o wide -o=custom-columns=NODE:.spec.nodeName --no-headers)
+
+	if [ ${CLUSTER_TYPE} == "minikube" ]; then
+		kubectl_cmd="kubectl -n monitoring"
+
+		MINIKUBE_IP=$(minikube ip)
+
+		KRUIZE_PORT=$(${kubectl_cmd} get svc kruize --no-headers -o=custom-columns=PORT:.spec.ports[*].nodePort)
+		KRUIZE_UI_PORT=$(${kubectl_cmd} get svc kruize-ui-nginx-service --no-headers -o=custom-columns=PORT:.spec.ports[*].nodePort)
+
+		export KRUIZE_URL="${MINIKUBE_IP}:${KRUIZE_PORT}"
+		export KRUIZE_UI_URL="${MINIKUBE_IP}:${KRUIZE_UI_PORT}"
+		export TECHEMPOWER_URL="${MINIKUBE_IP}:${TECHEMPOWER_PORT}"
+	elif [ ${CLUSTER_TYPE} == "openshift" ]; then
+		kubectl_cmd="kubectl -n openshift-tuning"
+
+		KRUIZE_IP=$(${kubectl_cmd} get pods -l=app=kruize -o wide -o=custom-columns=NODE:.spec.nodeName --no-headers)
+		KRUIZE_UI_IP=$(${kubectl_cmd} get pods -l=app=kruize-ui-nginx -o wide -o=custom-columns=NODE:.spec.nodeName --no-headers)
+
+		KRUIZE_PORT=$(${kubectl_cmd} get svc kruize --no-headers -o=custom-columns=PORT:.spec.ports[*].nodePort)
+		KRUIZE_UI_PORT=$(${kubectl_cmd} get svc kruize-ui-nginx-service --no-headers -o=custom-columns=PORT:.spec.ports[*].nodePort)
+
+		export KRUIZE_URL="${KRUIZE_IP}:${KRUIZE_PORT}"
+		export KRUIZE_UI_URL="${KRUIZE_UI_IP}:${KRUIZE_UI_PORT}"
+		export TECHEMPOWER_URL="${TECHEMPOWER_IP}:${TECHEMPOWER_PORT}"
+	fi
+
+	kruize_local
+	get_urls
+
+	end_time=$(get_date)
+	elapsed_time=$(time_diff "${start_time}" "${end_time}")
+	echo "Success! Kruize demo setup took ${elapsed_time} seconds"
+	echo
+	if [ ${prometheus} -eq 1 ]; then
+		expose_prometheus
+	fi
+}
+
+function kruize_local_demo_terminate() {
+	start_time=$(get_date)
+	echo
+	echo "#######################################"
+	echo "#       Kruize Demo Terminate       #"
+	echo "#######################################"
+	echo
+	kruize_uninstall
+	delete_repos autotune
+	if [ ${CLUSTER_TYPE} == "minikube" ]; then
+		minikube_delete
+	fi
+	end_time=$(get_date)
+	elapsed_time=$(time_diff "${start_time}" "${end_time}")
+	echo "Success! Kruize demo cleanup took ${elapsed_time} seconds"
+	echo
+}
+
+# Check if minikube exists and check system configs
+check_minikube
```

Comment:
This should be called when cluster type is minikube, otherwise on openshift it displays this 
```
./local_monitoring_demo.sh -c openshift -i quay.io/kruize/autotune_operator:0.0.21_mvp -u quay.io/kruize/kruize-ui:kldemo
ERROR: Please install minikube and try again
       Minikube resource config needed for demo:
       CPUs=8, Memory=16384MB
```